src/DynamoCore/Properties/Resources.en-US.resx

```diff
@@ -286,7 +286,9 @@ Sorry about that.</value>
     <value>Loading node definition for "{0}" from: {1}</value>
   </data>
   <data name="NodeProblemEncountered" xml:space="preserve">
```

Comment:
`NodeProblemEncountered` has a placeholder, as we remove it, shall we update its references to string concatenation?